packages/manager/modules/pci/src/projects/new/config/component.js

```diff
@@ -7,9 +7,15 @@ export default {
   template,
   bindings: {
     cart: '<',
-    summary: '<',
     getActionHref: '<',
     goToPayment: '<',
+    hds: '<',
+    hdsAddonOption: '<',
+    hdsFeatAvailable: '<',
+    isValidHdsSupportLevel: '<',
     model: '<',
+    ovhSubsidiary: '<',
```

Comment:
Still needed ? Maybe we can access `Environment.getUser()` directly ?

Reply:
No, i just missed it.
Fixed.